src/Modules/OrchardCore.Commerce.Inventory/Views/InventoryPart.Edit.cshtml

```diff
@@ -0,0 +1,19 @@
+@model InventoryPartViewModel
+
+<h3>@T["Inventories"]</h3>
+<div class="row">
+    @foreach (var inventoryKey in Model.Inventory.Keys)
+    {
+        var inventoryTitle = Model.Inventory.Count > 1 ? inventoryKey.Split('-').Last() : string.Empty;
```

Comment:
It would be reasonable to add a `SplitByDash` string extension instead of using `.Split('-')` like 8 times in the solution.

Reply:
The question is where to add this? `Split('-')` is used in 3 different projects, but none of these are suitable to place this extension method as you couldn't make use of it in all places. For example, if you create it in `OrchardCore.Commerce`, you cannot use it in `OrchardCore.Inventory` and `OrchardCore.MoneyDataType`.

To be honest, it's a simple and short enough call to just have it as is, but if you have a suggestion that'd allow such an extension method to be used in all solutions in the project in a simple manner, I'll add it.

Reply:
I find your argument sufficient to discard my request.